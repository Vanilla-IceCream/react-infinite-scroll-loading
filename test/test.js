import React from 'react';
import { render } from '@testing-library/react';

import InfiniteScroll from '~/';

process.chdir(__dirname);

describe('react-infinite-scroll-loading', () => {
  it('should render', () => {
    const loadMore = jest.fn();

    const children = (
      <div>
        <div className="item">1</div>
        <div className="item">2</div>
        <div className="item">3</div>
      </div>
    );

    const { container } = render(
      <div>
        <InfiniteScroll pageStart={0} loadMore={loadMore} hasMore={false}>
          <div className="list">{children}</div>
        </InfiniteScroll>
      </div>
    );

    expect(container).toMatchSnapshot();
  });
});
