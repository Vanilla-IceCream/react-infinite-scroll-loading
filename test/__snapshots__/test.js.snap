// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-infinite-scroll-loading should render 1`] = `
<div>
  <div>
    <div>
      <div
        class="list"
      >
        <div>
          <div
            class="item"
          >
            1
          </div>
          <div
            class="item"
          >
            2
          </div>
          <div
            class="item"
          >
            3
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
